### DISCUSSION TITLE
2007/09/29/%E7%AC%AC%E4%B8%80%E6%AC%A1%E7%BB%99%E5%AE%A2%E6%88%B7%E4%B8%8A%E8%AF%BE%E7%9A%84%E7%BB%8F%E5%8E%86

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#179] [Announcements]
# 2007/09/29/%E7%AC%AC%E4%B8%80%E6%AC%A1%E7%BB%99%E5%AE%A2%E6%88%B7%E4%B8%8A%E8%AF%BE%E7%9A%84%E7%BB%8F%E5%8E%86

两个星期前，我当了一次 LabVIEW 高级课程的教师。

https://blog.qizhen.xyz/2007/09/29/%E7%AC%AC%E4%B8%80%E6%AC%A1%E7%BB%99%E5%AE%A2%E6%88%B7%E4%B8%8A%E8%AF%BE%E7%9A%84%E7%BB%8F%E5%8E%86

**Comment by ruanqizhen** [comment]
之前博客下的留言：

Qizhen
2007年10月11日 11:36 编辑
正好看到一个网页，有NI虚拟实验室的合作计划：http://digital.ni.com/worldwide/china.nsf/web/all/9E2F0D3B42EEBB8348256C6E00274CBA?OpenDocument&node=200128_zhs

回复
Qizhen
2007年10月10日 08:25 编辑
红小薯我不太清楚你说的在网络环境下是指什么，不过我知道好多大学都已经建了虚拟实验室。前一阵子还在北京听了一些老师介绍他们的经验。我想你可以先联系那些已经建好的实验室，向他们寻求一些经验。三次方高级课程目前只开过一次，下一次开，不知要到什么时候。

回复
zhang
2007年10月7日 21:05 编辑
你好
看了你的很多文章，很是感慨，很是钦佩。
现在想上虚拟实验室，想请教下LabVIEW在网络环境下有无较好的解决办法或方案？
谢谢。

回复
梦
2007年10月6日 21:32 编辑
这些课程在北京开设么？

回复
YALU
2007年10月2日 20:52 编辑
最近怎样?
 
国庆假期都做什么呢?

回复
杰
2007年10月2日 15:44 编辑
我只听过basic的课程，呵呵。属于入门水平的。
最近准备去听下intermediate的课程，想考个cld的证书。不过据说不太好考。
想听你的课程还需要一段时间哦。
 

回复
Chien-Chi
2007年09月29日 22:34 编辑
我在台灣有上過Advanced的課程,內容真的是屬於高階的應用,我之前學的內容很少有OOP的課程,聽說課程內容已有更新,OOP也是我目前最想學習的課程

回复
Jing
2007年09月29日 16:52 编辑
赞，可惜没听到。

回复
